src/data/contexts/alertsPromptContext.scss

```diff
@@ -0,0 +1,76 @@
+.alert-card {
+  position: fixed;
+  pointer-events: all;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  border: 1px solid #ccc;
+  border-radius: 20px;
+  cursor: pointer;
+  min-width: 90%;
+  max-width: 500px;
+  padding: 10px 20px;
+  text-align: left;
+  transition: transform 0.3s ease, width 0.3s ease;
+  top: 75%;
+  left: 50%;
```

Comment:
It's better to use `bottom` instead of `top` if you want to align something based on the bottom of the screen. Using `top` will result in inconsistent results with different heights of screens.